tests/base/src/test/java/org/wildfly/security/sasl/oauth2/OAuth2SaslClientV11Test.java

```diff
@@ -620,4 +537,28 @@ private SaslClient createSaslClientFromConfiguration(URI serverUri) throws SaslE
         AuthenticationConfiguration authenticationConfiguration = contextConfigurationClient.getAuthenticationConfiguration(serverUri, context);
         return contextConfigurationClient.createSaslClient(serverUri, authenticationConfiguration, Collections.singletonList(SaslMechanismInformation.Names.OAUTHBEARER));
     }
+
+    private void testWithSaslClientAndServer(URI serverUri) throws Exception {
```

Comment:
Looks great @xjusko! One more potential simplification would be to update this method to take a `String serverUri` instead of a `URI` and then the `URI.create(serverUri)` call could be done in this method. That way each test would just need to call `testWithSaslClientAndServer` with a string.

Reply:
You are right, it should be fixed now.